website/shared/meta-cli/embedded-typegate.mdx

```diff
@@ -0,0 +1,16 @@
+
+If you have installed the _non-thin_ version of _Meta CLI_, it comes with an embedded typegate packaged inside it. The [typegate](/docs/reference/typegate) is where we deploy our [typegraphs](/docs/reference/typegraph) where any logic written in them is exposed via an HTTP or GraphQL endpoints. You can run an embedded typegate node from the terminal, but you first need to set a couple of environment variables which are needed to instantiate the typegate node. 
```

Comment:
I would assume that they have by default, and instead of highlight the non-thin, the thin flavor shall instead of the "exception"